docs/source/advanced/mixed_precision.rst

```diff
@@ -0,0 +1,76 @@
+.. testsetup:: *
+
+    from pytorch_lightning import Trainer
+
+
+.. _amp:
+
+Mixed Precision Training
+========================
+
+Mixed precision combines the use of both FP32 and lower bit floating points (such as FP16) to reduce memory footprint during model training, resulting in improved performance.
+
+Lightning offers mixed precision training for GPUs, CPUs and TPUs.
+
+.. note::
+
+    In some cases it is important to remain in FP32 for numerical stability, so keep this in mind when using mixed precision.
```

Comment:
I thought that's what mixed precision does? Only converting ops that are safe to do in fp16?

Or maybe I understood the note wrong? :D

Reply:
i know for torchpoint32 you have to remain in FP32 when doing scatter ops during computation. But that is as far as my knowledge goes :D

Reply:
I think the note is just trying to answer the question why we can't do true FP16 training in many cases.